### PR TITLE
visualizing tracking data - 1

### PR DESCRIPTION
**Pull request by Jonak-Adipta-Kalita**

### CODE REVIEW

File: flomo/cli.py [hunk 1]
```diff
@@ -39,5 +40,16 @@ def start(tag: str, name: str):
     ui.main(tag.lower(), name, session_id)
 
 
+@flomo.command(aliases=["t"])
+def tracking():
```
Comment (moiSentineL) [inline flomo/cli.py:44]: can add a command to reset the data (delete the sessions.db file and re-initialise_

File: flomo/tracker.py [hunk 2]
```diff
@@ -47,3 +49,18 @@ def get_sessions(self):
...
+
+def show_sessions():
+    db = Tracker()
+    sessions = db.get_sessions()
+    db.conn.close()
+
+    print(
+        tabulate.tabulate(
+            pandas.DataFrame(sessions),  # type: ignore
+            headers=["ID", "Tag", "Name", "Start Time", "End Time", "Total Time"],
```
Comment (moiSentineL) [inline flomo/tracker.py:62]: new headers and data:
```
|Session Date | Tag | Name | Total Time |
```

- Include time and date in session date.
- colorise tag based on `config.json`

**Review by moiSentineL — COMMENTED** [review]
works well, can improve

File: flomo/tracker.py (hunk 2 above)
Comment (Jonak-Adipta-Kalita) [reply flomo/tracker.py:63]: sure, also, do you want me to not store the `start_time` and `end_time` at all, or store em just dont show?

Comment (moiSentineL) [reply flomo/tracker.py:63]: > sure, also, do you want me to not store the `start_time` and `end_time` at all, or store em just dont show?

i think you need for the calculations and stuff, right? so just **don't show dem**

**Comment by Jonak-Adipta-Kalita** [comment]
@moiSentineL I will do the rest of the changes in a diff pr. you can review and merge it. the pagination will be done after the "rich ui" visualization.

**Pull request merged**